### PR TITLE
feat(swap): impl. swap from asset selection popup

### PR DESCRIPTION
**Pull request by nistadev**
image

WIP

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[Create the "From" selection popup, which displays the current available balance assets](https://emurgo.atlassian.net/browse/YOEXT-652)**

[](__JIRA_COMMENT_FLAG)

File: packages/yoroi-extension/app/components/swap/SelectAssetDialog.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
+import { useState } from 'react';
+import { Box, Typography } from '@mui/material';
+import { ReactComponent as AssetDefault } from '../../assets/images/revamp/asset-default.inline.svg';
+import { ReactComponent as NoAssetsFound } from '../../assets/images/revamp/no-assets-found.inline.svg';
+import { ReactComponent as SearchIcon } from '../../assets/images/revamp/icons/search.inline.svg';
+import { truncateAddressShort } from '../../utils/formatters';
+import Dialog from '../widgets/Dialog';
+
+export default function SelectAssetDialog({ assets = [], type, onAssetSelected, onClose }) {
```
Comment (neuodev) [inline packages/yoroi-extension/app/components/swap/SelectAssetDialog.js:9]:
```suggestion
export default function SelectAssetDialog({ assets = [], type, onAssetSelected, onClose }): Node {
```

File: packages/yoroi-extension/app/components/swap/SelectAssetDialog.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+            flexDirection="column"
+            gap="16px"
+            alignItems="center"
+            justifyContent="center"
+          >
+            <Box mt="60px">
+              <NoAssetsFound />
+            </Box>
+            <Typography variant="body1" fontWeight={500}>
+              No tpkens found for “{searchTerm}”
```
Comment (neuodev) [inline packages/yoroi-extension/app/components/swap/SelectAssetDialog.js:136]: Please use `Intl`
```suggestion
              No tokens found for “{searchTerm}”
```

**Comment by nistadev** [comment]
Closing in favor of: https://github.com/Emurgo/yoroi-frontend/pull/3321

**Pull request closed without merging**